packages/showcases/svelte/stories/components/structure/VtmnAccordion/VtmnAccordion.stories.svelte

```diff
@@ -0,0 +1,146 @@
+<script>
+  import { Meta, Story } from '@storybook/addon-svelte-csf';
+  import { VtmnAccordion, VtmnButton } from '@vtmn/svelte';
+  import { parameters } from '@vtmn/showcase-core/csf/components/structure/accordion.csf';
+  const argTypes = {
```

Comment:
Should be moved in `@vtmn/showcase-core/csf/components/structure/accordion.csf` as we did with other files.